Src/Idioms.FsCheck.FsCheck2UnitTest/Idioms.FsCheck.FsCheck2UnitTest.fsproj

```diff
@@ -20,10 +20,19 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FsCheck" Version="[2.0.1]" />
     <PackageReference Include="Unquote" Version="4.0.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='net452'">
+    <PackageReference Include="FSharp.Core" Version="[4.2.3]" PrivateAssets="All" />
```

Comment:
I found that we have a mix of "exact" (`[version]`) and "at least" (`version`) specified dependency versions. Is it on purpose? Could we just go with "exact" everywhere in unit tests?

Reply:
@zvirja I went for exact versions on `.NET 4.5` projects, to prevent NuGet from restoring a later version. I suppose we could use the minimum necessary version for each project.